### ISSUE TITLE
Dynamic filters don't work on secondary tabs.

### ISSUE THREAD

**Issue by MrJoy** [avo-hq/avo#1974]
### Describe the bug

When I have multiple filterable resources displayed as associations from a third resource, with each of the filterable resources being confined to a tab, then I cannot actually use the filters for anything but the first tab.

### Steps to Reproduce
1. Set up <https://github.com/MrJoy/avo-debug>
2. `bin/dev`
3. Navigate to `http://localhost:5000/admin/resources/users/1`
4. Click on the `Calendar Instances` tab
5. Attempt to add a filter
6. Notice that the `fields` network request is made, and returns what appears to be the correct result but nothing happens on the page.

### Expected behavior & Actual behavior

Filters should behave properly whether on an index view, an association view directly on a resource panel, or in any tab on a resource panel.

### Models and resource files

<https://github.com/MrJoy/avo-debug>

### System configuration
**Avo version**: 3.0.1.beta20

**Rails version**: 7.0.8

**Ruby version**: 3.2.2

**License type**:

 - [ ] Community
 - [ ] Pro
 - [x] Advanced

**Are you using Avo monkey patches, overriding views or view components?**

 - [ ] Yes. If so, please post code samples.
 - [x] No

#### Impact

 - [ ] High impact (It makes my app un-usable.)
 - [x] Medium impact (I'm annoyed, but I'll live.)
 - [ ] Low impact (It's really a tiny thing that I could live with.)

#### Urgency

 - [ ] High urgency (I can't continue development without it.)
 - [x] Medium-High urgency (I don't have a workaround, but I can continue development / operation for now -- however, it will become an issue once we have any sort of market traction)
 - [ ] Medium urgency (I found a workaround, but I'd love to have it fixed.)
 - [ ] Low urgency (It can wait. I just wanted you to know about it.)

**Comment by adrianthedev** [comment]
@Paul-Bob can you please check this one out?

**Comment by MrJoy** [comment]
@Paul-Bob This does not appear to be fixed as of beta22.  Is the fix going out in a subsequent version?

**Comment by Paul-Bob** [comment]
Yes @MrJoy it will be fixed on the next release. Thanks for the detailed report and for the reproduction repository!
